helix-view/src/theme.rs

```diff
@@ -14,15 +14,18 @@ use toml::{map::Map, Value};
 use crate::graphics::UnderlineStyle;
 pub use crate::graphics::{Color, Modifier, Style};
 
+// store default theme in to byte array.
+const DEFAULT_THEME_DATA: &[u8] = include_bytes!("../../theme.toml");
+const BASE16_DEFAULT_THEME_DATA: &[u8] = include_bytes!("../../base16_theme.toml");
```

Comment:
Is it possible to setup constants with these parsed as `toml::Value`s? That way we don't have to re-parse them when doing inherits

Reply:
I followed the existing code base to use ```Lazy<Value>``` to parsed it.